### PR TITLE
Add basic documentation

### PR DESCRIPTION
**Pull request by TomWFox**
- Adds SPM, cocoapods, Carthage installation instructions
- Adds contributing doc
- Changes copyright notice in license
- Improve readme design

### CODE REVIEW

**Comment by ShawnBaek** [comment]
@Tom-Fox1 Thank you for contributing. Who is maintain parse-swift repo? We need to update the contributing guide.

**Comment by dplewis** [comment]
There is no maintainer for this repo.

New maintainers are welcome

**Comment by cbaker6** [comment]
@TomWFox I think we can add some of the usage/testing from https://github.com/parse-community/Parse-Swift/pull/12#issuecomment-656918156, particularly the playgrounds part.

The project can be imported via SPM, Cocoapods, Carthage (I've never use this before, but the tests pass) by linking to the repo link directly

**Comment by cbaker6** [comment]
For the Contributing guide, we can add, `brew install swiftlint`

**Comment by TomWFox** [comment]
@cbaker6 @pranjalsatija I think this is good to go now.

The details in the contributing doc are very basic right now but at some point I'll add info about setting up a Parse Server locally to test etc.

Btw I haven't actually tried the SPM or Carthage install but I think they should both work.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+    <img alt="Parse Platform" src="logo large.png" width="200">
+  </a>
+</p>
 
-> There is currently no maintainer for this repository, new maintainers are welcome here or on the Parse Objective-c SDK
+<h2 align="center">ParseSwift</h2>
 
-[EXPERIMENTAL] Parse pure Swift SDK
+<p align="center">
+    An experimental pure Swift library that gives you access to the powerful Parse Server backend from your swift applications.
```
Comment (pranjalsatija) [inline README.md:9]: `swift applications` => `Swift applications`?

**Comment by cbaker6** [comment]
@TomWFox lgtm!

**Comment by pranjalsatija** [comment]
@TomWFox just confirming that I was able to install ParseSwift, then build and launch a blank iOS single view app template using SPM and CocoaPods with no issues.

**Comment by TomWFox** [comment]
@cbaker6 good point, added

**Comment by TomWFox** [comment]
@pranjalsatija thanks for checking, only Carthage that is untested now (by us anyway).

**Comment by pranjalsatija** [comment]
I just tested that out too (just to be safe), it worked.

**Comment by TomWFox** [comment]
Great!

**Pull request merged**